src/Microsoft.Azure.WebJobs.Logging/Entities/InstanceTableEntity.cs

```diff
@@ -90,9 +89,14 @@ internal static string RowKeyFunctionInstanceId(Guid functionInstanceId)
 
         public string TriggerReason { get; set; }
 
-        public DateTimeOffset StartTime { get; set; }
+        public DateTime StartTime { get; set; }
 
-        public DateTimeOffset? EndTime { get; set; }
+        public DateTime? EndTime { get; set; }
+
+        // Last heart beat. When EndTime is missing, this lets us guess if the function is still alive. 
+        // Whereas Start and End are written very determinsitically in the lifecycle, 
+        // heartbeat is special because it's written at potentially random times by a background poller thread. 
```

Comment:
This is the existing poller thread used to track some aggregate stats.  It's a task started when we emit our first start event; and it's stopped when we do the final flush. 

---
In reply to: [90490676](https://github.com/Azure/azure-webjobs-sdk/pull/928#discussion_r90490676) [](ancestors = 90490676)